applications/FreeSurfaceApplication/tests/generalTests.py

```diff
@@ -43,7 +43,7 @@ def ExecuteReservoirTests(self):
             self.checkResults()
 
     def setUp(self):
-        self.check_tolerance = 1e-6
+        self.check_tolerance = 1e-5
```

Comment:
@joaquinirazabal may I ask why you required to change the tolerance? I think the changes in this PR must not affect the results of the tests.

Reply:
My guess is that other changes in the code (FluidDynamics or Meshing applications) are causing the tests to fail. However, the differences are very small, and, in my opinion, a tolerance 1e-5 for the DISTANCE function is more than enough.

Reply:
Could be. I think it is reasonable too.